hal/lib/athena/PDP.cpp

```diff
@@ -6,14 +6,14 @@
 /*----------------------------------------------------------------------------*/
 
 #include "HAL/PDP.h"
+#include "ctre/PDP.h"
```

Comment:
It's a result of the way I check for "related headers". If a file with the same base name as the cpp file is included, it is considered related. I'm not sure how to disambiguate them.